### PR TITLE
Add action dropdown descriptions

### PR DESCRIPTION
**Pull request by rigelk**
*note*: descriptions will need to be rewritten. The way I wrote them is more as a placeholder to get feedback on the modification.

Screenshot_2020-01-15 Account video channels - PeerTube

### CODE REVIEW

**Comment by Chocobozzz** [comment]
I like it! Could you also add some icons to see how they are rendered?

File: client/src/app/+admin/config/edit-custom-config/edit-custom-config.component.html [hunk 1]
```diff
@@ -165,7 +165,7 @@
 
           <div class="form-group">
             <label i18n for="instanceBusinessModel">How will you finance the PeerTube server?</label>
-            <div class="label-small-info">With you own funds? With users donations? Advertising?</div>
+            <div class="label-small-info">With your own funds? With users donations? Advertising?</div>
```
Comment (Chocobozzz) [inline client/src/app/+admin/config/edit-custom-config/edit-custom-config.component.html]: we forgot a `i18n` here

File: client/src/app/+my-account/my-account-history/my-account-history.component.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="top-buttons">
   <div class="history-switch">
     <p-inputSwitch [(ngModel)]="videosHistoryEnabled" (ngModelChange)="onVideosHistoryChange()"></p-inputSwitch>
-    <label i18n>Enable video history</label>
+    <span class="ml-2 text-uppercase" i18n>Video history</span>
```
Comment (Chocobozzz) [inline client/src/app/+my-account/my-account-history/my-account-history.component.html]: I think we should keep a verb for checkboxes (enable seems okay in this case)

Comment (rigelk) [reply client/src/app/+my-account/my-account-history/my-account-history.component.html]: I don't think the "Enable" adds information.

**Comment by rigelk** [comment]
@Chocobozzz what do you mean?

**Comment by Chocobozzz** [comment]
> @Chocobozzz what do you mean? You want a screenshot of a situation where icons are involved?

Yes, to see how they are positioned

**Comment by rigelk** [comment]
Screenshot_2020-01-20 Account video channels - PeerTube

File: client/src/app/shared/buttons/action-dropdown.component.ts [hunk 3]
```diff
@@ -4,9 +4,12 @@ import { GlobalIconName } from '@app/shared/images/global-icon.component'
 export type DropdownAction<T> = {
   label?: string
   iconName?: GlobalIconName
+  description?: string
+  title?: string
   handler?: (a: T) => any
   linkBuilder?: (a: T) => (string | number)[]
   isDisplayed?: (a: T) => boolean
+  actions?: DropdownAction<T>[]
```
Comment (Chocobozzz) [inline client/src/app/shared/buttons/action-dropdown.component.ts]: where is it used?

Comment (rigelk) [reply client/src/app/shared/buttons/action-dropdown.component.ts]: it is not, I shouldn't have included it: that was just an artifact of me trying to have nested dropdowns.

Comment (rigelk) [reply client/src/app/shared/buttons/action-dropdown.component.ts]: it is not used. It should have been removed.

**Pull request merged**